From 521a1a0fb0ee5cd5e0df284ccfcc0e9931fef9aa Mon Sep 17 00:00:00 2001
From: Antonios Vamporakis <ant@area128.com>
Date: Thu, 9 Jan 2014 01:25:50 +0100
Subject: fixup! MIPS: add board support for ZyXEL P-2601HN-Fx

Add ethernet support.

Signed-off-by: Antonios Vamporakis <ant@area128.com>

diff --git a/board/zyxel/p2601hnfx/p2601hnfx.c b/board/zyxel/p2601hnfx/p2601hnfx.c
index 6a69af9..1b02fc4 100644
--- a/board/zyxel/p2601hnfx/p2601hnfx.c
+++ b/board/zyxel/p2601hnfx/p2601hnfx.c
@@ -26,7 +26,7 @@ int checkboard(void)
 
 static const struct ltq_eth_port_config eth_port_config[] = {
 	/* MAC0: REALTEK RTL8306 switch */
-	{ 0, 0x0, LTQ_ETH_PORT_SWITCH, PHY_INTERFACE_MODE_RMII },
+	{ 0, 0x0, LTQ_ETH_PORT_SWITCH, PHY_INTERFACE_MODE_RMII_OC },
 };
 
 static const struct ltq_eth_board_config eth_board_config = {
diff --git a/include/configs/p2601hnfx.h b/include/configs/p2601hnfx.h
index 5cba30f..31fd6e0 100644
--- a/include/configs/p2601hnfx.h
+++ b/include/configs/p2601hnfx.h
@@ -14,12 +14,18 @@
 /* Configure SoC */
 #define CONFIG_LTQ_SUPPORT_UART			/* Enable ASC and UART */
 
+#define CONFIG_LTQ_SUPPORT_ETHERNET		/* Enable ethernet */
+
 #define CONFIG_LTQ_SUPPORT_NOR_FLASH		/* Have a parallel NOR flash */
 
 #define CONFIG_LTQ_SUPPORT_SPL_NOR_FLASH	/* Build NOR flash SPL */
 #define CONFIG_LTQ_SPL_COMP_LZO			/* Compress SPL with LZO */
 #define CONFIG_LTQ_SPL_CONSOLE			/* Enable SPL console */
 
+/* Switch devices */
+#define CONFIG_SWITCH_MULTI
+#define CONFIG_SWITCH_RTL8306
+
 /* Environment */
 #if defined(CONFIG_SYS_BOOT_NOR)
 #define CONFIG_ENV_IS_IN_FLASH
@@ -62,6 +68,6 @@
 #define CONFIG_EXTRA_ENV_SETTINGS	\
 	CONFIG_ENV_LANTIQ_DEFAULTS	\
 	CONFIG_ENV_UPDATE_UBOOT_NOR	\
-	"kernel_addr=0xB0040000\0"
+	"kernel_addr=0xB0060000\0"
 
 #endif /* __CONFIG_H */
-- 
2.7.4

